test/views/MainTemplateViewSpec.scala

```diff
@@ -176,15 +176,27 @@ class MainTemplateViewSpec extends ViewSpec {
           val textOfScript: String = doc.getElementById("timeoutScript").html()
 
           textOfScript shouldBe
-            """$.timeoutDialog({timeout: 120, countdown: 30, """ +
-              """time: "eiliad", """ +
-              """title: "Rydych ar fin cael eich allgofnodi", """ +
-              """message: "Er eich diogelwch, byddwn yn eich allgofnodi cyn pen", """ +
-              """minute_text: "munud", """ +
-              """minutes_text: "o funudau", """ +
-              """keep_alive_button_text: "Ailddechrau eich sesiwn", """ +
-              """heading_text: "Rydych wedi bod yn anweithredol am sbel.", """ +
-              """keep_alive_url: '/lookup-address/renewSession',logout_url: '/lookup-address/destroySession?timeoutUrl=testTimeoutUrl'});var dialogOpen;"""
+            """$.timeoutDialog({
+              |    timeout: 120,
+              |    countdown: 120,
+              |    time: "eiliad",
+              |    title: "Rydych ar fin cael eich allgofnodi",
+              |    message: "Er eich diogelwch, byddwn yn eich allgofnodi cyn pen",
+              |    minute_text: "munud",
+              |    minutes_text: "o funudau",
+              |    keep_alive_button_text: "Ailddechrau eich sesiwn",
+              |    heading_text: "Rydych wedi bod yn anweithredol am sbel.",
+              |    keep_alive_url: "testTimeoutKeepAliveUrl",
+              |    logout_url: '/lookup-address/destroySession?timeoutUrl=testTimeoutUrl'
+              |});
+              |
+              |var dialogOpen;
+              |
+              |$(document).on("click", "#timeout-keep-signin-btn", function() {
+              |    // Renew the address-lookup session as well as the calling service
+              |    // Taken from https://github.com/hmrc/assets-frontend/blob/master/assets/patterns/help-users-when-we-time-them-out-of-a-service/timeoutDialog.js#L138
+              |    $.get('/lookup-address/renewSession', function () {})
+              |})"""
```

Comment:
This test is failing on whitespace differences. There must be a nicer way to test this but my scala-foo isn't up to scratch.